### PR TITLE
Fix candle updating before trigger

### PR DESCRIPTION
**Pull request by ivaaaan**

### CODE REVIEW

File: src/aggregation_rules/aligned_time_rule.rs
Comment (MathisWellmann) [inline src/aggregation_rules/aligned_time_rule.rs]:
```suggestion
        open: Open,
        close: Close,
        num_trades: NumTrades<u32>,
        volume: Volume,
```
Looks like `high` and `low` are not used in the test.

**Review by MathisWellmann — DISMISSED** [review]
This makes sense.
The reason I changed it in https://github.com/MathisWellmann/trade_aggregation-rs/commit/ff766c09970fce40f34527a628d9549b3e9f428f 
was related to `RelativePriceRule`, but it was not correct to make those changes.
Thanks for this PR.

**Comment by MathisWellmann** [comment]
@ivaaaan you can remove the changed images in `img` if you like. That folder should now be ignored by `.gitignore`.

**Comment by ivaaaan** [comment]
@MathisWellmann done, I have removed images and unused properties from test

**Review by MathisWellmann — DISMISSED** [review]
Pending CI checks, LGTM

**Comment by MathisWellmann** [comment]
I'll publish a new version to crates.io after your other PR #17 is merged.

**Pull request merged**
